### DISCUSSION TITLE
How should planters be represented in OSM?

### DISCUSSION THREAD

**Discussion by Robinlovelace** [dabreegster/abstreet#712] [Q&A] [answered]
This may not be the ideal place for this discussion but it's a discussion not an issue and started in #660 in this repo so figured here is as good as anywhere (maybe the OSM wiki talk pages would be better but couldn't figure out how to use that).

In any case, here's the question, should `barrier=planter` *really* be restricted to nodes only? Asking because the [OSM Wiki page](https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dplanter) on the topic had them down as node only. I changed that to enable them to represent ways, as this way in Leeds is linear feature of 7/8 planters: https://www.openstreetmap.org/way/965986113

It seems that more than 1/3 of existing `barrier=planter` tags are already linear features.

Just wanted to check with people experienced with OSM, e.g. @matkoniecz and @mvl22, that I've not made a mistake as I'm quite new to OSM wiki editing (started yesterday)! Mission creep would be enabling them to be areas and relations I guess but nodes and ways seem most relevant to me intuitively and based on what I've seen in real life.

**Comment by matkoniecz** [comment, marked as answer]
Yes, that makes perfect sense! I have not considered this case when copying the template, thanks for fixing it.

https://en.wikipedia.org/wiki/Help:Talk_pages may help as we use very similar software at OSM Wiki as English Wikipedia.

I will freely admit that talk pages at wiki are a horrific hack and not how interface for a discussion should work.

**Reply by matkoniecz** [reply]
Note that in general OSM Wiki describes situation, may contain advice/suggestion/reasoning for preferring some way of mapping but does not rule.

Anyone is free to map contrary to OSM Wiki statements (but note that going against community consensus is a bad idea in general, and OSM Wiki is quite good at capturing it - but there are many, many things at OSM Wiki that are wrong/misleading/incomplete)

**Reply by Robinlovelace** [reply]
Fantastic, thanks for the quick, comprehensive and I guess final answer. Nodes and Ways it is then :+1: FYI I posted links to the new pages and now probably a few more people will see them after the London Cycling Campaign, which has 40k+ followers, retweeted the links: https://twitter.com/London_Cycling/status/1417870927971471368

**Reply by tyrasd** [reply]
FYI: The overview page (https://wiki.openstreetmap.org/wiki/Key:barrier#Values) still lists `barrier=planter` as a node-only barrier. And it should probably also be moved one section down to _Linear barriers or access control on highways_.

**Reply by tyrasd** [reply]
Another PS:

Allowing `barrier=planter` on lines makes the tag slight incompatible with the `man_made=planter` tag, which is kind of a recommended tag combination by the wiki page:

> One object can be tagged both with [man_made](https://wiki.openstreetmap.org/wiki/Key:man_made)=[planter](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dplanter) and [barrier](https://wiki.openstreetmap.org/wiki/Key:barrier)=planter

But since `man_made=planter` is only allowed on points and areas, the combination cannot be used on linear planter barriers.

See also https://github.com/openstreetmap/id-tagging-schema/pull/506#issuecomment-1159577378

**Reply by Robinlovelace** [reply]
Thanks for this @tyrasd, very glad to have your input here! In terms of next steps, do you recommend me or others go in and edit the OSM wiki? A bit new to the community so very thankful for your input (and hi from Leeds)!
